db_mutex/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-from .db_mutex import db_mutex
```

Comment:
sorry, i can update the docs. any package that was importing a model in the init had to be changed during the django 1.9 upgrade so that models are not imported before django initializes all apps.